alienfile

```diff
@@ -3,10 +3,26 @@
 use alienfile;
 
 plugin 'PkgConfig' => (
-    pkg_name        => [qw/ tidy tidy5 tidy-html5 /],
+    pkg_name        => 'tidy',
```

Comment:
using an array like this is intended for packages like libgmp that provide multiple .pc files, not for packages that might have differently names .pc files.  I don't think this packages is distributed with differently named .pc files though, `tidy-htm5` is the FreeBSD package name, but the .pc file for it is still `tidy.pc`.